packages/fiori/cypress/specs/NotificationList.cy.tsx

```diff
@@ -56,6 +57,600 @@ function Sample() {
 	</NotificationList>;
 }
 
+describe("Notification List Item Tests", () => {
+	it("tests itemToggle fired", () => {
+		const stub = cy.stub().as("myStub");
+		cy.mount(<NotificationListGroupItem id="nlgi1" onClick={stub} />);
+
+		cy.get("#nlgi1")
+			.shadow()
+			.find(".ui5-nli-group-header")
+			.realClick();
+
+		cy.get("@myStub").should("have.been.calledOnce");
+	});
+
+	it("tests Group List Header keyboard shortcuts ", () => {
+		cy.mount(
+			<NotificationList id="notificationList">
+				<NotificationListGroupItem id="nlgi3" collapsed>
+					<NotificationListItem id="nli5" />
+				</NotificationListGroupItem>
+			</NotificationList>
+		);
+
+		cy.get("#nlgi3")
+			.shadow()
+			.find(".ui5-nli-group-items")
+			.should("not.be.visible");
+		cy.get("#nlgi3").realClick();
+		cy.get("#nlgi3").should("be.focused");
+
+		cy.get("#nlgi3")
+			.shadow()
+			.find(".ui5-nli-group-items")
+			.should("be.visible");
+		cy.get("#nlgi3").realPress("ArrowLeft");
+
+		cy.get("#nlgi3")
+			.shadow()
+			.find(".ui5-nli-group-items")
+			.should("have.css", "display", 'none');
+		cy.get("#nlgi3").realPress("ArrowRight");
+
+		cy.get("#nlgi3")
+			.shadow()
+			.find(".ui5-nli-group-items")
+			.should("have.css", "display", 'block');
+		cy.get("#nlgi3").realPress("-");
+
+		cy.get("#nlgi3")
+			.shadow()
+			.find(".ui5-nli-group-items")
+			.should("have.css", "display", 'none');
+		cy.get("#nlgi3").realPress("+");
+
+		cy.get("#nlgi3")
+			.shadow()
+			.find(".ui5-nli-group-items")
+			.should("have.css", "display", 'block');
+		cy.get("#nlgi3").realPress("-");
+		cy.get("#nlgi3").realPress("Space");
+
+		cy.get("#nlgi3")
+			.shadow()
+			.find(".ui5-nli-group-items")
+			.should("have.css", "display", 'block');
+	});
+
+	// Notification List Item specific tests follows
+
+	it("tests itemClick fired", () => {
+		const itemClick = cy.stub().as("myStub");
+		cy.mount(<NotificationListItem id="nli1" onClick={itemClick} />);
```

Comment:
Wrap the item with NotificationList and attach `item-click` handler`. Remove this click handler